packages/xstate-wallet/src/workflows/ethereum-enable.ts

```diff
@@ -97,15 +97,15 @@ export const ethereumEnableWorkflow = (
     hideUi: () => {
       messagingService.sendDisplayMessage('Hide');
     },
-    sendResponse: async (context: WorkflowContext, event) => {
+    sendResponse: async (context: WorkflowContext) => {
       messagingService.sendResponse(context.requestId, {
         signingAddress: await store.getAddress(),
         walletVersion: WALLET_VERSION,
         selectedAddress: context.enabledAddress as string
-      }); // TODO: typing
```

Comment:
Is this still a TODO, but untracked? Or is this now done?

Reply:
No, it was fixed (the arguments are typed), but the TODO was never removed.